src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -332,7 +322,7 @@ export const TradeQuote: FC<TradeQuoteProps> = memo(
         bodyContent={bodyContent}
         onClick={handleQuoteSelection}
         isActive={isActive}
-        isActionable={isTradingActive && hasAmountWithPositiveReceive && errors.length === 0}
```

Comment:
If trading is halted we'll have at least one error:

<img width="469" alt="Screenshot 2024-06-19 at 11 23 45 AM" src="https://github.com/shapeshift/web/assets/97164662/e7d4353a-2aa5-40fa-9f73-5ea577beee23">

Reply:
🧠